lib/matplotlib/axis.py

```diff
@@ -1320,7 +1320,13 @@ def get_minor_formatter(self):
         return self.minor.formatter
 
     def get_major_ticks(self, numticks=None):
-        """Get the tick instances; grow as necessary."""
+        """
+        Get the tick instances; grow as necessary.
```

Comment:
The "grow as necessary" part is an implementation detail which is irrelevant for the the user and rather confusing. I'd remove that. Then we can simply

~~~
r"""Return the list of major `.Tick`\s."""
~~~